test/Microsoft.AspNet.Mvc.TagHelpers.Test/InputTagHelperTest.cs

```diff
@@ -685,6 +842,19 @@ private class Model
             public NestedModel NestedModel { get; set; }
 
             public bool IsACar { get; set; }
+
+            [DataType(DataType.Date)]
+            public DateTime Date { get; set; }
```

Comment:
Maybe name this `DateWithDataType` or something along those lines so it's easier to read as part of `MemberData`?